### ISSUE TITLE
New User Improvements: Update Sample Data

### ISSUE THREAD

**Issue by laurenliz22** [SEED-platform/seed#2595]
Update current sample data to allow new users to go through SEED's more notable features as a supplement to the tutorial.  Include easy-to-use column mapping with the sample data

Data:
- Use Alex's advanced features list (see data from https://seed-platform.org/references/ under "Presentations".  There is a zip file with a directory for each feature to try out
- Maintain existing simple dataset
- Provide data sufficient to running analysis for Bsyncr (Adrian has an example dataset he has been using for the Bsyncr analysis) or BETTER

Pre-populate new user accounts with the data:
- If someone has no Properties then have them "click here" to populate with sample data
- Option to reset the current organization (this is currently captured in another ticket)

**Comment by laurenliz22** [comment]
@RDmitchell - The following New User Improvement tickets are suggestions our team developed to enhance the experience for new SEED Platform users.  Before we start working through these tickets to update SEED we want to get your sign off on the changes.  Would you mind looking at the descriptions and letting me know if you have any questions or changes.  Thanks! Lauren

**Comment by RDmitchell** [comment]
I don't quite understand all the nuances of the data suggestions, but I'm sure you will come up with a good solution.

**Comment by RDmitchell** [comment]
@macintoshpie / @axelstudios -- should I test this on a "public instance" ? Is there such an instance? 

And if not, should I just test on dev1, with a new user account?

**Comment by macintoshpie** [comment]
@RDmitchell testing on dev1 should be sufficient, you can create a new account or new organization to test.

**Comment by RDmitchell** [comment]
instance: dev1
SHA: 
Org: LBNL 444
Cycle: 2020 Calendar Year (default)

I created a new user and org, then upon logging in I clicked on the "Auto-Populate Sample Data"

This screen appeared
![image](https://user-images.githubusercontent.com/6314950/128071866-2b13f116-b83c-41b1-bfba-2e78529a8419.png)

I clicked the Continue button and the program populated the org with the data

**Comment by RDmitchell** [comment]
Here are my comments about the sample data -- I can make separate issues for these if that is useful, and close this issue.

* Geocoding
  * There are actually 3 properties that show up on the map -- the one left out of the description in the screenshot above is PM Property ID # 5766973; Seems like it should be added to the list
![image](https://user-images.githubusercontent.com/6314950/128093993-6739b8d3-3bdd-4c4c-ba0b-cb52850ece26.png)

* Merging
  * This example is a bit weird because everything in the two files is exactly the same, including the Created date and time, except for the one field of Site EUI, which are different. But I would have thought that line would be highlighted in yellow because the two values are different. 
![image](https://user-images.githubusercontent.com/6314950/128094260-77f6d8f0-c640-4ccd-ab2e-9a02072a403c.png)
  * There is not a filename at the top of each column. A better example would be to have imported 2 different files, with two different creation dates, and possibly 2 different filenames, which would then show the updates of the 1st file from the 2nd.
![image](https://user-images.githubusercontent.com/6314950/128094321-8bc92432-6f75-4f7f-b0b7-182e003187a7.png)

* Tax Lots
  * the example does show a connected record between tax lot and property
  * It would be useful to add two more examples: 
    * 2 tax lots associated with 1 PM Property ID, and
    * 2 PM property IDs associated with 1 Tax Lot
![image](https://user-images.githubusercontent.com/6314950/128094526-0c9cb42b-b8db-4062-bf4a-f2e150b04c82.png)

* Labels
  * The labels are not being displayed when I click on the label column -- this seems like a bug, so I will make an issue for this
 ![image](https://user-images.githubusercontent.com/6314950/128094668-b63b09eb-b419-4041-992a-dc1575384a9e.png)
 
* BSyncr Analysis
  * The instructions should add something about which BSycnr Model Selection option to pick
![image](https://user-images.githubusercontent.com/6314950/128094768-d101a0f0-8f27-4b87-a75e-954baaeb1ac0.png)
  * After running the analysis, it doesn't seem to show any results, although I don't actually know what should be displayed there
![image](https://user-images.githubusercontent.com/6314950/128094835-08fc7834-1585-4b37-961c-e44f6cfabaaa.png)

**Comment by RDmitchell** [comment]
@macintoshpie -- let me know what you want to do with these comments. I can make separate issues for each of them, some of which would I guess be "Features" and a few which might be "bugs". And then maybe I should close this issue, since the basic functionality of the auto populated data appears to work.

**Comment by macintoshpie** [comment]
@RDmitchell I like your idea of making new issues to address features / bugs. Closing the issue now since the initial work seems to be functional. Thanks!
